web/gds-user-ui/src/modules/dashboard/member/utils.ts

```diff
@@ -39,25 +39,25 @@ export const getVaspNetwork = (dir: any) => {
   }
 };
 
-export const getVaspStatus = (status: number) => {
+export const getVaspStatus = (status: any) => {
   switch (status) {
-    case 1:
+    case 1 || 'NO_VERIFICATION':
```

Comment:
I added strings here to be consistent with `getBusinessCategory` below. Once the BE fix is made, this can be updated.